src/GraphQL/Types/QueryArguments.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace GraphQL.Types
 {
+    // TODO: better to rename QueryArguments and QueryArgument to something like FieldArguments and FieldArgument thought directives have arguments too.
```

Comment:
How about `GraphQLArgument`?  It appears that is what the JavaScript project calls it.

Reply:
Regardless of the name, if we do rename these classes in 4.0, I would maintain backwards compatibility, marking the old ones as obsolete.  There are going to be a LOT of references in user code, and it cannot be fixed with just a namespace import.

Reply:
Maybe. Either way we will be thinking abput it only for 4.0 release, not now.

Reply:
GraphQLArgument seems to be obvious good candidate.

Reply:
If they remain and are marked as obsolete but are no longer used, they will still be of little use. All the same, the client will not be able to work until he changes all his code. Or I didn't understand your intentions.

Reply:
I meant to leave stubs in there
```csharp
[Obsolete]
public class QueryArgument : GraphQLArgument { }
```
Of course with any necessary constructors that previously existed.

Reply:
👍 👍 👍